wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/ConversationMessageSectionController.swift

```diff
@@ -274,17 +274,44 @@ final class ConversationMessageSectionController: NSObject, ZMMessageObserver {
             return false
         }
 
-        return context.isLastMessage || selected || message.deliveryState == .failedToSend || message.hasReactions()
+        return context.isLastMessage || message.deliveryState == .failedToSend || message.isSentBySelfUser
```

Comment:
Is the edited timestamp part of the toolbox view?

Reply:
Yes!